indico/cli/util.py

```diff
@@ -112,3 +112,7 @@ def get_usage(self, ctx):
 
     def get_params(self, ctx):
         return self._impl.get_params(ctx)
+
+
+def format_event(event):
+    return f'- {event.id}: [{event.created_dt.strftime("%Y-%m-%d")}] {event.title} ({event.external_url})'
```

Comment:
`start_dt`

Reply:
and you can use `.date()` on it; using a date object in a string context will automatically use its isoformat